test/src/travis-dep-builder.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+#
+#
+
+prefix=$HOME/local
+cachedir=$HOME/local/.cache
+
+#
+# Ordered list of software to download and install into $prefix.
+#  NOTE: Code currently assumes .tar.gz suffix...
+#
+downloads="\
+https://www.open-mpi.org/software/ompi/v2.0/downloads/openmpi-2.0.2.tar.gz"
+
+declare -A extra_configure_opts=(\
+)
+
+declare -r prog=${0##*/}
+declare -r long_opts="prefix:,cachedir:,verbose,printenv"
+declare -r short_opts="vp:c:P"
+declare -r usage="\
+\n
+Usage: $prog [OPTIONS]\n\
+Download and install to a local prefix (default=$prefix) dependencies\n\
+for building flux-framework/flux-core\n\
```

Comment:
Somehow, my provious comment is gone... hmmm. Any case, please change "flux-framework/flux-core" to "LaunchMON"